### PR TITLE
[WPB-11973] Freeze API version 7, create dev API version 8.

### PR DESCRIPTION
**Pull request by fisx**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow https://docs.wire.com/developer/developer/api-versioning.html#version-bump-checklist
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by fisx** [comment]
some of the current failures:

```
----- Events.testConsumeEventsAckNewEventWithoutAckingOldOne FAIL (10.07 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:176


----- Events.testConsumeEventsAcks FAIL (10.06 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:127


----- Events.testChannelLimit FAIL (12.79 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:321

3. k at test/Testlib/ModService.hs:52

4. withModifiedBackend at test/Test/Events.hs:302


----- Events.testConsumeEventsForDifferentUsers FAIL (10.15 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:79

3. assertClientAdd at test/Test/Events.hs:74


----- Events.testConsumeEventsMultipleAcks FAIL (10.08 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:152


----- Events.testConsumeEventsOneWebSocket FAIL (10.06 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:36


----- Events.testConsumeEventsWhileHavingLegacyClients FAIL (10.12 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:110


----- Events.testEventsDeadLettered FAIL (13.04 s) -----
assertion failure:
No event received for 1s

call stack:
1. assertFailure at test/Test/Events.hs:418

2. assertEvent at test/Test/Events.hs:218

3. k at test/Testlib/ModService.hs:52

4. withModifiedBackend at test/Test/Events.hs:203

```

File: services/brig/src/Brig/API/User.hs [hunk 1]
```diff
@@ -325,6 +325,8 @@ createUser ::
 createUser new = do
   email <- fetchAndValidateEmail new
 
+  -- lift $ liftSem $ blockListInsert (fromJust (error "bämmmm") email)
```
Comment (stefanwire) [inline services/brig/src/Brig/API/User.hs:328]: Left-overs?

Comment (fisx) [reply services/brig/src/Brig/API/User.hs]: gone!

File: services/brig/src/Brig/API/Public.hs [hunk 2]
```diff
@@ -452,14 +453,16 @@ servantSitemap =
 
     userClientAPI :: ServerT UserClientAPI (Handler r)
     userClientAPI =
-      Named @"add-client-v6" addClient
```
[the remark this answers is not in the document]
Comment (stefanwire) [reply services/brig/src/Brig/API/Public.hs:455]: V6 names are not finalised yet, since they only become finalised with v7. This should be fine. 🔥🐶☕

**Comment by fisx** [comment]
looks good to me! if somebody can review for the green mark, we're ok to merge.

**Review by stefanwire — APPROVED** [review]
Would be better with one more independent review.

**Comment by fisx** [comment]
talked to stefan and we agreed we're good!

**Pull request merged**